src/components/Modals/FilterModal/FilterModalContent/FilterModalContent.tsx

```diff
@@ -57,4 +58,11 @@ const FilterModalContent = ({
     );
 };
 
+const formatTestId = (text: string) => {
```

Comment:
add this to format-value.ts and reference it from there.

Reply:
This has been added and reference given in the .tsx file it is been called to.